integration-tests/runtime-tests/src/generators/exampleGenerators/testTransactionGenerator.ts

```diff
@@ -0,0 +1,40 @@
+import { ApiPromise, Keyring, WsProvider } from '@polkadot/api';
+import { KeyringPair } from '@polkadot/keyring/types';
+import * as R from 'ramda';
+import minimist from 'minimist';
+import { args } from '../../utils/args';
+
+
+/**
+ * Generates a test transaction.
+ * This is mainly an example on how to add a new generator.
+**/
+export class testTransactionGenerator {
+  /**
+   * Sends test transaction from Alice to Bob.
+   * @param {ApiPromise} api Connected API Promise.
+  **/
+  public static async testTransaction(api: ApiPromise, walletSender, walletReceiverAddress) {
+    const transfer = api.tx.assets.transferNative(walletReceiverAddress, 12345678910, true);
+    const hash = await transfer.signAndSend(walletSender, { nonce: -1 });
```

Comment:
What effect has the negative nonce?

Reply:
See https://polkadot.js.org/docs/api/cookbook/tx/#how-do-i-take-the-pending-tx-pool-into-account-in-my-nonce

Reply:
ah nice convenience feature, but shame that they are using magic values, typical JS lol